### ISSUE TITLE
question

### ISSUE THREAD

**Issue by jackfrost168** [yuto3o/SKNet-Keras#1]
In the paper, the Select part uses A and B as different branchs soft attention, where is your A and B?
After x = layers.Conv2D(filters*M, 1, name=name+'_fc_x')(z), you directly make the output is filter*M?

**Comment by yuto3o** [comment]
In my implementation,  the soft attention for  different branchs is generated at the same time (e.g.  the length for each branch is 'filters', and we have 'M' branches).  And then I reshape the output for different branch(from [batch, 1, 1, filters*M] to [batch, 1, 1, filters, M]).  After softmax, soft attention for different branches could be calculated in one matrix. I think such implementation is more efficient.   
"""
x = layers.Conv2D(filters*M, 1, name=name+'_fc_x')(z)
x = layers.Reshape([1, 1, filters, M],name=name+'_reshape')(x)
scale = layers.Softmax(name=name+'_softmax')(x)
"""

**Comment by jackfrost168** [comment]
thank you for your answer！
